Add Countdown article (#3194)
ppy/osu-wiki: wiki/Beatmapping/Countdown/en.md

---
tags:

>     - Timer

Reviewer: ```suggestion
  - Timer
```

---

# Countdown

*For tournament timers, see: [Tournament Management Commands](/wiki/osu!tourney/Tournament_Management_Commands)*

> *For regulations around countdowns, see: [Ranking Criteria](/wiki/Ranking_Criteria)*

Reviewer: ```suggestion

*For regulations around countdowns, see: [Ranking Criteria](/wiki/Ranking_Criteria)*
```

> Countdowns give players a countdown before starting a [beatmap](/wiki/Beatmaps). This is useful for songs that go directly into the heat of things. Most maps do not need this option enabled. The option can be disabled in the [song setup](/wiki/Beatmap_Editor/Song_Setup) menu, under the "Design" tab. Countdowns are not allowed in osu!taiko.

Reviewer: Not sure.

```suggestion
A countdown may appear in the beginning of a [beatmap](/wiki/Beatmaps). This is useful for songs that go directly into the heat of things. Most maps do not need this option enabled. The option can be disabled in the [song setup](/wiki/Beatmap_Editor/Song_Setup) menu, under the "Design" tab. Countdowns are not allowed in osu!taiko.
```

> When enabled, the countdown animation can be refined by using the `Countdown Speed` buttons (`Half`, `Normal`, `Double`) and the `Countdown Offset` button. `Countdown Speed` buttons will halve, double or keep the standard value of the countdown animation. `Countdown Offset` is the offset in beats before the first hit object. This means that a countdown with countdown offset 2 will have its countdown animation start 2 beats earlier than normal. For maps with intros, skipping will go directly to the countdown.

Reviewer 2: ```suggestion
When enabled, the countdown animation can be controlled using the `Countdown Speed` buttons (`Half`, `Normal`, `Double`) and the `Countdown Offset` button. `Countdown Speed` buttons halve, double or keep the standard speed of the countdown animation. `Countdown Offset` sets the offset in beats before the first hit object. For example, a countdown with offset 2 has its animation start 2 beats earlier than normal. For maps with intros, skipping goes directly to the countdown.
```
present tense and some other wording
